openedx/core/djangoapps/profile_images/tests/helpers.py

```diff
@@ -0,0 +1,48 @@
+"""
+Helper methods for use in profile image tests.
+"""
+import os
+from tempfile import NamedTemporaryFile
+
+from django.core.files.uploadedfile import UploadedFile
+from PIL import Image
+
+
+
+def make_image_file(dimensions=(320, 240), extension=".jpeg", force_size=None):
+    """
+    Returns a named temporary file created with the specified image type and options.
+
+    Note the default dimensions are unequal (not a square) ensuring that center-square
+    cropping logic will be exercised during tests.
+    """
+    image = Image.new('RGB', dimensions, "green")
+    image_file = NamedTemporaryFile(suffix=extension)
```

Comment:
Make sure these get cleaned up - according to the docs they have to be `.close()`d for that to happen. I'm not seeing that you're manually closing these, so the only other possibility is that one of the libraries is doing that for you.